### PR TITLE
Updating parallelization arguments of gtfs2gps()

### PR DESCRIPTION
**Pull request by pedro-andrade-inpe**
- argument `strategy` is now deprecated
- it will use a `future::plan()` automatically if previously set

### CODE REVIEW

File: R/gtfs2gps.R [hunk 1]
```diff
@@ -3,17 +3,22 @@
...
-#' distance, indicated as a spatial resolution. This function uses progressr
-#' internally to show progress bars.
+#' distance, indicated as a spatial resolution. It is possible to use future package
+#' to parallelize the execution (or use argument plan). This function also
+#' uses progressr internally to show progress bars.
 #' 
 #' @param gtfs_data A path to a GTFS file to be converted to GPS, or a GTFS data
 #' represented as a list of data.tables.
 #' @param spatial_resolution The spatial resolution in meters. Default is 50m.
-#' @param parallel Decides whether the function should run in parallel. Defaults to FALSE.
```
Comment (rafapereirabr) [inline R/gtfs2gps.R:12]: Hi Pedro. Can we can keep the parameter `parallel` ? It is much more intuitive than `plan`

**Review by rafapereirabr — COMMENTED** [review]
Hi Pedro. Can we can keep the parameter parallel ? It is much more intuitive than plan

File: R/gtfs2gps.R (hunk 1 above)
Comment (pedro-andrade-inpe) [reply R/gtfs2gps.R:12]: Do you mean parallel instead of plan? Sure.

Comment (rafapereirabr) [reply R/gtfs2gps.R:12]: yes, thanks!

**Review by rafapereirabr — APPROVED** [review]

**Pull request merged**